services/121-service/src/payments/payments.controller.ts

```diff
@@ -89,6 +89,31 @@ export class PaymentsController {
     return await this.paymentsService.getProgramPaymentsStatus(programId);
   }
 
+  @AuthenticatedUser({ permissions: [PermissionEnum.PaymentREAD] })
```

Comment:
do I understand correctly that this new endpoint is to be used in Portalicious only then and not in Portal?

Reply:
Yes correct, unless it becomes necessary in the Portal (but that would have to be an explicit request which is unlikely).